hack/tools/controller-gen/go.mod

```diff
@@ -0,0 +1,38 @@
+module github.com/giantswarm/apiextensions/hack/tools/code-generator
```

Comment:
This module name feels wrong, maybe we change it to something simpler like: 

```suggestion
module code-generator
```

Reply:
File deleted, with k8sapi flavour we don't need these files anymore :+1: